models/user.dao.js

```diff
@@ -0,0 +1,52 @@
+const { AppDataSource } = require("./dataSource");
+
+  const createuser = async (email, password, firstname, lastname, skintype) => {
```

Comment:
JavaScript 언어의 컨벤션 중 하나는 camelCase를 지키는 것입니다.
camelCase는 띄어쓰기 혹은 언더바(_)로 단어를 구분하는 것이 아니라 대문자로 띄어쓰는 부분을 구분하는 작성 방법입니다.
```suggestion
  const createUser = async (email, password, firstname, lastname, skintype) => {
```